packages/react-table/src/utils/useFirstRender/index.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from "react";
+
+export const useFirstRender = () => {
```

Comment:
what do you think ? 🤔 

```suggestion
export const useIsFirstRender = () => {
```